dspace-xmlui/src/main/webapp/sitemap.xmap

```diff
@@ -542,8 +541,12 @@
 
                     <!-- OpenSearch results document for whole repo or community/collection -->
                     <map:match pattern="open-search/**">
+                       <!-- Prior to DSpace 6.0, "open-search/" was mapped to the StandardOpenSearchGenerator,
+                            which was removed as part of removal of the legacy Lucene search index classes.
+                            This pattern has been removed to DiscoveryOpenSearchGenerator to help ensure
+                            minimal impact from removal of the legacy Lucene search index. -->
                         <map:match pattern="open-search/">
-                            <map:generate type="StandardOpenSearchGenerator"/>
+                            <map:generate type="DiscoveryOpenSearchGenerator"/>
```

Comment:
NOTE: this PR also fixes [DS-2268](https://jira.duraspace.org/browse/DS-2268) as the old OpenSearch generator is now replaced by the Discovery driven one.